pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -136,6 +138,8 @@ def _kokkos_configuration():
         "QFT",
         "ECR",
         "BlockEncode",
+        "MidMeasureMP",
+        "Conditional",
```

Comment:
These are not regular PL operations and this will break Catalyst. We should remove them from here and use class-instance comparison in the decomposition preprocessing step to preserve them in the tape. Please check my fix for LQ in PR #665 to see how these ops were removed from the list. 
```suggestion
```